src/DynamicRupture/Output/OutputManager.cpp

```diff
@@ -326,6 +314,45 @@ void OutputManager::initPickpointOutput() {
     }
   }
 
+  std::stringstream baseHeader;
+
+  auto suffix = [&allReceiversInOneFilePerRank](auto pointIndex, auto simIndex) {
+    std::string suffix;
+
+    if (allReceiversInOneFilePerRank) {
+      suffix += std::to_string(pointIndex);
+    }
+
+    if constexpr (seissol::multisim::MultisimEnabled) {
+      suffix += std::to_string(simIndex);
+    }
+
+    return suffix;
```

Comment:
It might be good, even though that'll change the format again by a bit, to separate between digits; e.g. by a `-`. Suppose you have a lot of receivers (200 per rank) and fuse 16 simulations. Then, what would `quantity1010` refer to? Either simulation `10`, aggregated receiver `10`; or simulation `0`, in receiver `101`.